mlrun/projects/project.py

```diff
@@ -2116,6 +2116,7 @@ def enable_model_monitoring(
         *,
         deploy_histogram_data_drift_app: bool = True,
         wait_for_deployment: bool = False,
+        overwrite: bool = False,
```

Comment:
Please add the overwrite argument to the `deploy_histogram_data_drift_app` function.

Reply:
I think ׳e can add it in different pr, not sure we want to add this option to the user.